src/Compilers/VisualBasic/Portable/Analysis/FlowAnalysis/RegionReachableWalker.vb

```diff
@@ -25,7 +25,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                     endPointIsReachable = If(walker._regionEndPointIsReachable.HasValue, walker._regionEndPointIsReachable.Value, walker.State.Alive)
                 Else
                     startPointIsReachable = True
-                    startPointIsReachable = False
+                    endPointIsReachable = False
```

Comment:
actually, it looks like the caller sets both of these to true if there is a failure.  so perhaps this should be `endPointIsReachable = true`?